eq-author/src/App/page/Design/Validation/DateValidation/index.js

```diff
@@ -65,15 +79,23 @@ const UnwrappedDateValidation = ({ validation, answer, displayName, onChange, on
           <ConnectedPath />
         </Column>
         <Column cols={END_COL_SIZE}>
-          {hasError && <StyledError>{ERR_NO_VALUE}</StyledError>}
+          {hasDurationError && <StyledError>{ERR_NO_VALUE}</StyledError>}
         </Column>
       </Grid>
       <Grid>
         <AlignedColumn cols={START_COL_SIZE}>
-          {answer.type === DATE
-            ? <PositionPicker value={validation.relativePosition} onChange={onChange} onUpdate={onUpdate} />
-            : <RelativePositionText> {validation.relativePosition.toLowerCase()} </RelativePositionText>
-          }
+          {answer.type === DATE ? (
+            <PositionPicker
+              value={validation.relativePosition}
+              onChange={onChange}
+              onUpdate={onUpdate}
+            />
+          ) : (
+            <RelativePositionText>
+              {" "}
+              {validation.relativePosition.toLowerCase()}{" "}
+            </RelativePositionText>
```

Comment:
Are the `{" "}` for spaces?

Reply:
No, the linter inserts them for some reason - I keep removing them and they go back :-(

Reply:
There we go - hopefully they're gone for good now